s4e-backend/src/main/java/pl/cyfronet/s4e/util/GeometryUtil.java

```diff
@@ -38,7 +38,7 @@ public Geometry parseWKT(String wkt, GeometryFactory factory) throws ParseExcept
     }
 
     public Coordinate[] closeCoords(Coordinate[] coords) {
-        if (coords == null || coords.length < 3 || coords[0].equals(coords[coords.length-1])) {
+        if (coords == null || coords.length < 3 || coords[0].equals(coords[coords.length - 1])) {
```

Comment:
1. Is it `x`, `y` or `z` coordinate, `coords[0]`? Is it polar, euclides plane? 
2. What does it mean `coords[0].equals(coords[coords.length-1])`? What kind of value is it?
3. What does mean `3` in this statement `coords.length < 3`?

Reply:
uh, i just auto-formatted this file :(